keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -618,32 +595,30 @@ feature.
 NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
 -->
 
-Yes. This will result in the timestamp value being eventually set to `nil`. More details in
-"Upgrade / Downgrade Strategy" section.
+Yes, for PVs not updated while feature was enabled. However once the `LastPhaseTransitionTime` value is set, disabling
+feature gate will not remove the value.
+
+More details in "Upgrade / Downgrade Strategy" section.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-No issues expected. There are three cases that can occur for a PV:
+No issues expected. There are two cases that can occur for a PV:
 
   1. PV did not transition its phase when feature gate was enabled - the `LastPhaseTransitionTime` field was not added 
      to the PV object so this is the same case as enabling the feature gate for the first time.
-  2. PV did transition its phase but the `LastPhaseTransitionTime` *was not* reset to `nil` because the PV was not 
-     validated when the feature was enabled - the timestamp value will be updated on next phase change as if the feature
-     gate was never disabled.
-  2. PV did transition its phase but the `LastPhaseTransitionTime` *was* reset to `nil` because the PV was validated at
-     least once already while feature was enabled - the timestamp value will be updated on next phase change because
-     updates from `nil` are allowed.
+
+  2. PV did transition its phase when feature gate was enabled - the `LastPhaseTransitionTime` field is already set,
+     and it's timestamp value will be updated on next phase change.
 
 See "Upgrade / Downgrade Strategy" and "Notes/Constraints/Caveats" sections for more details.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Unit tests for enabling and disabling feature gate will be added when transitioning to beta. See "Graduation criteria"
-section.
+Unit tests for enabling and disabling feature gate will be added in alpha - see "Graduation criteria" section.
```

Comment:
Were the tests added?

In fact this comment from the KEP template was introduced for exactly the cases like this:
```
Additionally, for features that are introducing a new API field, unit tests that
are exercising the `switch` of feature gate itself (what happens if I disable a
feature gate after having objects written with the new field) are also critical.
You can take a look at one potential example of such test in:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
```